omnigibson/envs/env_base.py

```diff
@@ -244,15 +264,10 @@ def _load_robots(self):
                     cls_type_descriptor="robot",
                 )
                 # Import the robot into the simulator
-                og.sim.import_object(robot)
-                robot.set_position_orientation(position=position, orientation=orientation)
-
-            if len(self.robots_config) > 0:
-                # Auto-initialize all robots
-                og.sim.play()
-                self.scene.reset()
-                self.scene.update_initial_state()
-                og.sim.stop()
+                self.scene.add_object(robot)
+                robot.set_local_pose(position=position, orientation=orientation)
```

Comment:
presumably because it's setting the robot wrt to its own environment scene (which would have a global offset). The sim is stopped so this is setting the robot base prim, presumably

Reply:
Yup. But the local pose sometimes might be something really irrelevant based on sim playing etc. so we should fix this. @hang-yin please add a TODO here to use scene_local_position_orientation API and create a ticket to create a get/set_scene_local_position_orientation API. Starter issue for Frank.